internal/lua/module.go

```diff
@@ -364,6 +399,10 @@ func (m *Module) tryPacketUnlock(dst string) {
 	}
 }
 
+func (m *Module) filterVXProtoErrors(err error) bool {
+	return errors.Is(err, vxproto.ErrTopicUnreachable)
```

Comment:
It would be nice to have a short description which tells why some errors need to be "filtered out".